README.md

```diff
@@ -59,14 +59,32 @@ The Dynamoose Changelog can be found in the [CHANGELOG.md](//github.com/dynamoos
 
 The Dynamoose Roadmap can be found in the [ROADMAP.md](//github.com/dynamoosejs/dynamoose/blob/master/ROADMAP.md) file. Help is always appreciated on these items. If you are able to help submit a PR so we can review and improve Dynamoose!
 
-## Development
+## Development Scripts
 
 Dynamoose has a few helpers for local development. These are included in the scripts section of our `package.json`.
 
-## npm test
+### npm test
 
-This will run all of our tests via grunt. This spins up a local DynamoDB Local instance, runs Mocha, and closes the DynamoDB Local instance at completion.
+This will run all of our tests via a node script. This spins up a local DynamoDB Local instance, runs Mocha, and closes the DynamoDB Local instance at completion.
 
-## npm run lint
+### npm run test:debug
+
+Same as above, but launched with `--inspect`. This will allow you to place debugger statements for working in the tests.
+
+### npm run test:debug:brk
+
+Same as above, but launched with `--inspect-brk`. This will allow you to place debugger statements for working in the tests, and not initialize the tests until you manually start it.
+
+### npm run lint
 
 This will run our lint configuration against all of our source files and alert you to any issues.
+
+### npm run ci
+
+This will run our lint, test, and coverage scripts. This is exactly what travis runs, if you'd like to verify prior to commit.
+
+### husky
```

Comment:
do you see this readme as only for users, not contributors? i've always treated these as a place for all about what goes on in the package, and the contributing document as how to meaningfully add back to the project. i see that you've made the pull request template more or less a contrib doc. 

i feel like explaining husky here is a good idea, i've had people ask why the couldn't push stuff up to a project, or why the commit just ins't working, so i've made a habit of being explicit. can we keep this or is a blocker for you?

Reply:
@hweeks As it reads right now, it's very confusing under that section. It makes it sound like it's a command contributors need to know regarding the project.

I think there are still some general things related to the project that could be explained in a contributing document that aren't in the PR template.

For now, I think we should move this to another section and have it read something like:

```
Other information

This project uses husky to run our lint prior to commit, and full CI prior to push. If you are having trouble committing or pushing please be sure your code passes the linter and our full CI test suite.
```

Reply:
i'm going to change it per your request. as long as my fear is noted and viewed as prolly not vaild i'm cool with killing it.

Reply:
I'm curious how husky shows error messages and such. If it is explicit enough it might be worth removing this.

I'm not sure I'm 100% sold on husky yet. I push a lot of times. Having to wait a minute or so to push might really hinder my workflow on this project. Or it could be really helpful. We will see how it goes tho.

Reply:
that's super fair, run it like this to avoid it if you'd like

`git commit -m 'whatever' --no-verify`

Reply:
@hweeks Doesn't work too well with Git GUI's haha.